pkg/version/version.go

```diff
@@ -0,0 +1,5 @@
+package version
+
+var (
+	BuildInfo        = "v0.7"
```

Comment:
If this version number is going to live in a new file, we will also need to update `scripts/prepare-for-release`. That script syncs the version numbers across all relevant files (currently only the README) whenever we cut new releases.

Reply:
BuildInfo just a placeholder value, it will get overwritten during build time by ldflags. I probably need more help in understanding the release cut process

```
EMBED_FLAGS=-ldflags '-X "${SIMPLE_EC2_CLOUDFORMATION_TEMPLATE_VAR}=${SIMPLE_EC2_CLOUDFORMATION_TEMPLATE_ENCODED}"\
 -X "${E2E_CFN_TEST_CLOUDFORMATION_TEMPLATE_VAR}=${E2E_CFN_TEST_CLOUDFORMATION_TEMPLATE_ENCODED}"\
 -X "${E2E_CONNECT_TEST_CLOUDFORMATION_TEMPLATE_VAR}=${E2E_CONNECT_TEST_CLOUDFORMATION_TEMPLATE_ENCODED}"\
 -X "${E2E_EC2HELPER_TEST_CLOUDFORMATION_TEMPLATE_VAR}=${E2E_EC2HELPER_TEST_CLOUDFORMATION_TEMPLATE_ENCODED}"\
 -X "${VERSION_VAR}=${VERSION}"'
```

Reply:
Ah, that makes sense. I'm okay with this design, but I'd suggest we use a more generic placeholder in this file so that future developers or build systems don't confuse it for an actual version number. Let's also add a comment indicating that the Makefile will fill it in, similar to [how the CloudFormation variables are handled.](https://github.com/awslabs/aws-simple-ec2-cli/blob/main/pkg/cfn/template.go#L20)

The way our release process works would be irrelevant at that point, but the main idea is that we have a script that creates a git tag for the new version number, updates it in any relevant human- or machine-readable portions of the repo, and then opens a PR for it on GitHub.

Reply:
Are you okey with called the place holder variable named "BUILD_VERSION_VAR" instead of "VERSION_VAR"?

Reply:
Thanks for your feedback @snay2 . Pushed with the above mentioned changes. Please review

Reply:
Thanks @icarthick!

I should clarify my earlier comment. I think we should use a generic **value** in the `BuildInfo` variable. The name of the variable is fine, but perhaps the value should be something like `VERSION_PLACEHOLDER`. The idea is that if we have a hardcoded version number like `v0.7`, it will quickly become out of date, and it may be confusing later on. If instead we have a value for the placeholder that is very obviously not a valid version number, it will be easier to understand the intent, as well as to identify if and when that placeholder ever leaks somewhere it shouldn't be.

So in addition to the other changes you've already made, can you change this line to the following? After that, I am satisfied with this PR.

```go
BuildInfo = "VERSION_PLACEHOLDER"
```

Reply:
Clearly, I overthought your earlier comment :) Latest commit, should have the change. Let me know if you are able to merge it